### PR TITLE
Issue 1320 characterization filtred results

### PR DESCRIPTION
**Pull request by AntonStepanof**
fixes https://github.com/OHDSI/Atlas/issues/1320

### CODE REVIEW

File: js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.html [hunk 1]
```diff
@@ -25,6 +25,19 @@ <h2 data-bind="css: classes('title')">
     <div data-bind="css: classes('filter')">
         <visualizations-filter-panel params="{ filterList: $component.filterList, live: true }"/>
     </div>
+    <div class="input-group" style="width: 30%">
```
Comment (pavgra) [inline js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.html]: No inline styles, please. Use classes

File: js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.html [hunk 2]
```diff
@@ -25,6 +25,19 @@ <h2 data-bind="css: classes('title')">
...
+        <h4>Threshold Level:</h4>
+        <div style="display: flex">
+            <input type="number" class="form-control" style="margin-right: 10px" placeholder="Threshold value"
```
Comment (pavgra) [inline js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.html]: No inline styles

File: js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.js [hunk 3]
```diff
@@ -132,6 +135,14 @@ define([
             return momentAPI.formatDateTimeUTC(date);
         }
 
+        updateThreshold() {
+            this.loadData();
+        }
+
+        resultCountText() {
+            return 'Viewing First ' + this.resultsCountFiltered() + ' of ' + this.resultsCount() + ' records';
```
Comment (pavgra) [inline js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.js]: Why not templated string?
`` `Viewing First ${this.resultsCountFiltered()} of ${this.resultsCount()} records` ``

File: js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.html (hunk 1 above)
Comment (AntonStepanof) [reply js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.html]: updated to classes

File: js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.html (hunk 2 above)
Comment (AntonStepanof) [reply js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.html]: updated to classes

File: js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.js (hunk 3 above)
Comment (AntonStepanof) [reply js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.js]: done

**Comment by pavgra** [comment]
Closed in favor of https://github.com/OHDSI/Atlas/pull/1982

**Pull request closed without merging**